tests/drivers/bbram/stm32.conf

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_COUNTER=y
+CONFIG_BBRAM=y
```

Comment:
Not blocking (not related to your PR): This test case implementation should eventually be refactored to use a common prj.conf.
Created https://github.com/zephyrproject-rtos/zephyr/issues/52754 to get it fix.